### PR TITLE
[15.0][ADD]website_product_attribute_description

### PR DESCRIPTION
**Pull request by dsolanki-initos**
Depends on https://github.com/OCA/product-attribute/pull/1071

### CODE REVIEW

**Comment by dsolanki-initos** [comment]
@MiquelRForgeFlow Can you please review this PR?

**Comment by dsolanki-initos** [comment]
@Tardo Can you please review this PR?

**Comment by dsolanki-initos** [comment]
@eLBati Can you please review this PR?

**Comment by eLBati** [comment]
@dsolanki-initos sorry, I don't know this module and I don't use v15

**Review by ikapasi-initos — APPROVED** [review]
Code Review and Functional test.
LGTM

**Comment by dsolanki-initos** [comment]
@cristinamartinrod Could you please review this PR?

File: website_product_attribute_description/static/src/css/tooltip.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
+.tooltip-inner {
```
Comment (tarteo) [inline website_product_attribute_description/static/src/css/tooltip.css:1]: Can you please use theme variables here (SCSS).

File: website_product_attribute_description/static/src/js/tooltip.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+odoo.define("product_attribute_description.product_attribute_description", function () {
```
Comment (tarteo) [inline website_product_attribute_description/static/src/js/tooltip.js:1]: Can you tighten up the selectors here, this is prone to conflict with other modules.

**Review by tarteo — CHANGES_REQUESTED** [review]
Thanks! Can you move this PR to oca/e-commerce?

**Comment by dsolanki-initos** [comment]
@tarteo, Closing this PR as this module is moved to OCA/e-commerce, PR: https://github.com/OCA/e-commerce/pull/731

**Pull request closed without merging**